fft_conv_pytorch/fft_conv.py

```diff
@@ -83,7 +84,7 @@ def fft_conv(
     dilation_ = to_ntuple(dilation, n=n)
 
     # internal dilation offsets
-    offset = torch.zeros(1, 1, *dilation_)
+    offset = torch.zeros(1, 1, *dilation_).to(signal.device)
```

Comment:
For efficiency, can you initialize directly on the device?
```python
offset = torch.zeros(1, 1, *dilation_, device=signal.device, dtype=signal.dtype)
```

^^ The `dtype` argument wasn't being used before, but I'm thinking that's a good idea while we're at it?

Reply:
Done.